documentation-site/pages/blog/responsive-web/index.mdx

```diff
@@ -0,0 +1,248 @@
+    paddingRight={['scale800', 'scale1200']}
+    {...props}
+  />
+);
+
+// is equivalent to
+
+const NavBar = themedStyled('div', ({$theme}) => ({
+  height: $theme.sizing.scale1400,
+  paddingRight: $theme.sizing.scale800,
+  paddingLeft: $theme.sizing.scale800,
+  [$theme.media.small]: {
+    height: $theme.sizing.scale1600,
+    paddingRight: $theme.sizing.scale1200,
+    paddingLeft: $theme.sizing.scale1200,
+  },
+}));
+```
+
+<BlogImage
+  src={responsiveNavBarGif}
+  alt="resizing the window smaller and seeing the yellow-highlighted nav bar change height and padding"
+  caption="NavBar component, highlighted in yellow, changes height and padding on mobile"
+/>
+
+## Mobile-only or desktop-only Blocks
+
+We can also take advantage of these responsive properties to completely hide a Block component using CSS `display: none`.
+
+```js
+// NavLink appears only on desktop
+const NavLink = props => <Block display={['none', 'block']} {...props} />;
+
+// TabLink appears only on desktop
+const TabLink = props => <Block display={['block', 'none']} {...props} />;
+```
+
+<BlogImage
+  src={responsiveHideGif}
+  alt="resizing the window smaller and seeing the green-highlighted components appear or disappear"
+  caption="Mobile/desktop-only components, highlighted in green, appear or disappear on mobile"
+/>
+
+## FlexGrid component
+
+Another handy tool for building responsive UIs is the [FlexGrid](/components/flex-grid/) component ([v6.19.0](https://github.com/uber-web/baseui/releases/tag/v6.19.0)+). This component lets you build grid layouts using CSS flexbox, supporting Internet Explorer 10+ users. It sets gaps between the columns and rows using `:nth-child` and `:nth-last-child` math, and also allows for responsive prop arrays like the [Block](/components/block/#responsive-layouts) component.
+
+```js
+// Example usage
+const Filters = () => (
+  <FlexGrid columnCount={[2, 4]} columnGap="scale300" rowGap="scale800">
+    <FlexGridItem>All Shipments</FlexGridItem>
+    <FlexGridItem>In Progress</FlexGridItem>
+    <FlexGridItem>Upcoming</FlexGridItem>
+    <FlexGridItem>Past</FlexGridItem>
+  </FlexGrid>
+);
+
+const Facilities = (Pickup, Dropoff) => (
+  <FlexGrid columnCount={[1, 2]} columnGap="scale300" rowGap="scale300">
+    <FlexGridItem>
+      <Pickup />
+    </FlexGridItem>
+    <FlexGridItem>
+      <Dropoff />
+    </FlexGridItem>
+  </FlexGrid>
+);
+```
+
+<BlogImage
+  src={responsiveDashboardGif}
+  alt="resizing the window smaller and seeing the red-highlighted grid components change column-counts"
+  caption="FlexGrid components, highlighted in red, change column-counts on mobile"
+/>
+
+## AspectRatioBox component
+
+On slower network connections, especially apparent on mobile networks, images can cause content to reflow as they are downloaded. When browsers see an image tag, they typically do not know how much space to allocate for it until it begins downloading the image headers, which can take some time.
```

Comment:
`content to reflow` sounds very technical, I would also expand with something that describes it from the user's point of view like `content jumps around`.